smartsim/_core/_cli/build.py

```diff
@@ -67,8 +67,6 @@ def check_py_onnx_version(versions: Versioner) -> None:
             f"Python {py_version.major}.{py_version.minor}. "
             "Instead consider using Python 3.8 or 3.9 with Onnx "
         )
-        if sys.platform == "linux":
-            msg += "1.2.5 or "
```

Comment:
Your removal makes sense but I'm confused by the underlying error message.  Shouldn't we be saying that 1.2.7 applies to the redisAI version not the ONNX version?

Reply:
yeah, that is a typo that I just noticed and asked Al to fix up while he is here

Reply:
Yep, see other comment on modified file - I am a bit confused.